src/components/Form.js

```diff
@@ -51,57 +53,58 @@ export default function Form({ handleData }) {
           {...register('foodName', {
             required: {
               value: true,
-              message: 'Wie willst Du etwas wiederfinden ohne Namen?',
+              message:
+                'How do you want to find something again without a name?',
             },
             maxLength: {
               value: 20,
-              message: 'ups, Limit von 2o Buchstaben erreicht',
+              message: 'ups, limit of 2o letters reached',
             },
             pattern: {
-              value: /^[A-Za-z]+/i,
-              message: 'bitte Text eingeben',
+              value: /^[A-Za-z ]+$/,
+              message: 'please enter text',
             },
           })}
         />
         {errors.foodName && <span>{errors.foodName.message}</span>}
-        <label htmlFor="foodTaste">Sorte *</label>
+        <label htmlFor="foodTaste">variety/taste *</label>
         <TextField
           id="foodTaste"
           maxLength="85"
           rows="4"
-          placeholder="z.B. Huhn, Lachs, Rind"
+          placeholder="z.B. chicken, salmon"
           aria-invalid={errors.foodTaste ? 'true' : 'false'}
           {...register('foodTaste', {
             required: {
               value: true,
-              message: 'Ohne diese Angabe kann die App nicht funktionieren.',
+              message: 'The app cannot function without this information.',
             },
             maxLength: {
               value: 84,
-              message: '84 Buchstaben sollten doch reichen.',
+              message: '84 letters should be enough.',
```

Comment:
see above, I would use "characters" instead of letters

Reply:
Done